packages/@tailwindcss-node/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@tailwindcss/node",
```

Comment:
Could this be an internal package like some of our others or do we need to actually publish it?

Reply:
I think it needs to be because of how the import works otherwise we'll have to duplicate these files into each package + configure the exports (which is something we _could_ do — I just think it's *probably* a worse tradeoff)